src/topology/order.lean

```diff
@@ -82,13 +82,23 @@ section lattice
 
 variables {α : Type u} {β : Type v}
 
-instance : partial_order (topological_space α) :=
+/- The following partial order will be convenient in order to get a complete
+   lattice instance via the Galois insertion method, but the partial order
+   that will be used elsewhere is the dual one, defined at the end of this
+   section. -/
+
+def tmp_order : partial_order (topological_space α) :=
 { le          := λt s, t.is_open ≤ s.is_open,
   le_antisymm := assume t s h₁ h₂, topological_space_eq $ le_antisymm h₁ h₂,
   le_refl     := assume t, le_refl t.is_open,
   le_trans    := assume a b c h₁ h₂, @le_trans _ _ a.is_open b.is_open c.is_open h₁ h₂ }
 
-lemma generate_from_le_iff_subset_is_open {g : set (set α)} {t : topological_space α} :
+local attribute [instance] tmp_order
+
+-- the following name looks wrong but will become correct at the end of the section when
+-- using the permanent partial order
+
+lemma le_generate_from_iff_subset_is_open {g : set (set α)} {t : topological_space α} :
```

Comment:
This should be a private/auxiliary lemma, and it should be restated with the correct le

Reply:
Is it better now?